de.variantsync.core/src/main/java/de/variantsync/core/interfaces/Grammar.java

```diff
@@ -0,0 +1,10 @@
+package de.variantsync.core.interfaces;
+
+public interface Grammar<T> {
+	enum OptionalType {
+		NodeOptional, TreeOptional, Mandatory
```

Comment:
We have to rename these but that is not your fault but a recent advance in science! (well actually not)

NodeOptional -> Wrapper
TreeOptional -> Optional
Mandatory remains unchanged